src/zoom.ts

```diff
@@ -43,71 +44,43 @@ export async function getZoomInfo(): Promise<ZoomInfo> {
 	});
 
 	const active = meetingInfo.status === 'started';
-
-	if (!active) {
-		hasSeenStart = true;
-		participants = [];
-	}
-
-	if (!hasSeenStart) participants = [];
-
-	return { meetingInfo, active, hasSeenStart, participants };
-}
-
-export async function updateDiscordStatusFromZoom(discord: Client, zoomInfo?: ZoomInfo) {
-	if (!zoomInfo) zoomInfo = await getZoomInfo();
-
-	if (zoomInfo.active) {
-		// Curently, the outer ternary is redundant because status is only set when zoomInfo.active is true
-		const status = zoomInfo.active
-			? hasSeenStart && participants.length >= 0
-				? `with ${
-						participants.length === 1 ? `1 person` : `${participants.length} people`
-				  } on Zoom`
-				: 'on Zoom'
-			: 'with nobody'; /* currently disabled */
-
-		console.log(`Setting Discord status to: ${status}`);
-		await discord.user?.setActivity(status, { type: 'PLAYING' });
-	} else {
-		console.log('Clearing Discord Status');
-		// This is the best way I can find to clear activity (bots cant have custom statuses)
-		await discord.user?.setActivity('', { type: 'CUSTOM_STATUS' });
+	const wasActive = getCallIsActive(getState());
+	// status change
+	if (wasActive !== active) {
+		if (active) {
+			dispatch(callStarted);
+		} else {
+			dispatch(callEnded);
+		}
 	}
+	return meetingInfo.join_url;
 }
 
-export async function processWebhookEvent(discord: Client, event: ZoomEvent) {
+export async function processWebhookEvent(discord: Client,  event: ZoomEvent) {
 	console.log('Processing Zoom Webhook Event:');
 	console.log(JSON.stringify(event, null, 2));
 
+	const { id: meeting } = event.payload.object;
```

Comment:
nit: I'm fine if you want to destructure this, but calling it `meeting` is confusing. I think it would be fine to do `const meeting = event.payload.object`, or call it `meetingId`.